crates/uv/src/commands/reporters.rs

```diff
@@ -154,15 +175,36 @@ impl ProgressReporter {
                 .unwrap()
                 .progress_chars("--"),
             );
+            // If the download is larger than 1MB, show a message to indicate that this may take
+            // a while keeping the log concise.
+            if multi_progress.is_hidden() && !*HAS_UV_TEST_NO_CLI_PROGRESS && size > 1024 * 1024 {
```

Comment:
Hmm, it's a bit strange that we hide these but _always_ show them if we don't have a size? I'd expect them to be always visible.

Reply:
I don't quite follow your comment. You expect them not to be gated by size? (I sort of agree, since it's not in a tty anyway it seems fine to write them all) Or you expect them to not be gated by the multiprogress status?

Reply:
I expect them _not_ to be gated by size, i.e., always shown. (In the `else` branch, where we don't _have_ a size, we always show them, so that discrepancy seemed odd to me.)

Reply:
Let's track in https://github.com/astral-sh/uv/issues/11272